app/views/gettingStarted.scala.html

```diff
@@ -1,40 +1,109 @@
-
-            <p>After choosing an example from <a href="//developer.lightbend.com/start/?group=play" target="_blank" rel="noopener noreferrer">Tech Hub</a> follow these steps:</p>
-            <ol>
-                <li>Click CREATE A PROJECT FOR ME to download the zipped project.</li>
-                <li>Unzip the project in a convenient location.</li>
-                <li>In a command window, navigate to the top level project directory.</li>
-                <li>Enter one of the following commands:
-                    <ul>
-                        <li>On OSx or Linux systems: <code>./sbt run</code> or <code>./gradle runPlayBinary</code></li>
-                        <li>On Windows systems: <code>sbt.bat run</code> or <code>gradlew.bat runPlayBinary</code></li>
-                    </ul>
-                    <p>The build tool downloads dependencies and compiles the project.</p>
-                </li>
-                <li>
-                    After the build finishes, enter the following URL in a browser to view the app:
-                    <p><a href="http://localhost:9000"><code>http://localhost:9000/</code></a></p>
-                </li>
-            </ol>
-            <p>Check out the readme file in the top level project directory to learn more about the example.</p>
+
+            <div class="try-section">
+                <h2>New to Play?</h2>
+                <p>Start with <a href="@controllers.documentation.ReverseRouter.latest(None)">our documentation</a> or the <a href="@controllers.documentation.ReverseRouter.latest(None, "HelloWorldTutorial")">Hello World Tutorial</a>.</p>
+
+                <div class="try-option">
+                    <h3><span>&raquo;</span> Learn whether Play Framework is right for your project</h3>
+                    <div class="try-option-content">
+                        <ul>
+                            <li><a href="@controllers.documentation.ReverseRouter.latest(None, "Introduction")">Introduction to Play</a></li>
+                            <li><a href="@controllers.documentation.ReverseRouter.latest(None, "Requirements")">Requirements to work With Play</a></li>
+                            <li><a href="@controllers.documentation.ReverseRouter.latest(None, "Philosophy")">Our philosophy</a></li>
+                        </ul>
+                    </div>
+                </div>
+
+                <div class="try-option">
+                    <h3><span>&raquo;</span> Follow the Hello World Tutorial</h3>
+                    <div class="try-option-content">
+                        <p>Play requires Java SE 1.8 or higher. You can check your version with the <code>java -version</code> command or install it from <a href="//www.oracle.com/technetwork/java/javase/downloads/index.html" rel="noopener">Oracle's site</a>. See details about the <a href="@controllers.documentation.ReverseRouter.latest(None, "Requirements")">requirements to work with Play</a>.</p>
```

Comment:
Do you think it would be easier for beginners to understand if we call it Java 8 rather than Java 1.8?